### DISCUSSION TITLE
Can we get some documentation on how to use the separate.py module in our own program? Does anyone have experience with this?

### DISCUSSION THREAD

**Discussion by maad-maxx** [Anjok07/ultimatevocalremovergui#519] [Q&A]
Title.

**Comment by Nafeij** [comment]
+1 to this. An API endpoint to download and use the MDX-Net models would be especially useful.

@maad-maxx Not sure if this helps, but in the meantime, you can take a look at the model-specific ones for [Demucs](https://github.com/facebookresearch/demucs/blob/main/demucs/separate.py) and [MDX-B](https://github.com/NaJeongMo/Colab-for-MDX_B/blob/main/main.py) and adapt them into API endpoints.

**Comment by ghost** [comment]
[#359 ](https://github.com/Anjok07/ultimatevocalremovergui/issues/359#issuecomment-1367843655)
